src/modules/localization/interface/src/main.cpp

```diff
@@ -0,0 +1,12 @@
+#include <localization_publisher.hpp>
+
+
+int main(int argc, char * argv[])
+{
+  auto measurements = read_data();
```

Comment:
Can we read this data inside `EKFPublisher` constructor?
This will keep the main.cpp file clean.

In future if we decide on getting all the sensor data by subscribing to a topic, we dont need to touch the main.cpp file at all.

Reply:
Sure.

Reply:
Yes that is fine since it will go away anyways. Resolving this comment.